blocks/api/test/parser.js

```diff
@@ -97,11 +98,16 @@ describe( 'block parser', () => {
 		it( 'returns false is block is not valid', () => {
 			registerBlockType( 'core/test-block', defaultBlockSettings );
 
+			const env = process.env.NODE_ENV;
+			process.env.NODE_ENV = 'production';
+
 			expect( isValidBlock(
 				'Apples',
 				getBlockType( 'core/test-block' ),
 				{ fruit: 'Bananas' }
 			) ).toBe( false );
+
+			process.env.NODE_ENV = env;
```

Comment:
Per above https://github.com/WordPress/gutenberg/pull/2056#discussion_r131038937, this now tests that `isValidBlock` throws.